.gitignore

```diff
@@ -0,0 +1,147 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# Custom ignored
+gpt3_config.yml
```

Comment:
Can you remove this and other entries that seem specific to your setup?

Reply:
removed irrelevant entries